portfolio/portfolio.go

```diff
@@ -444,3 +444,39 @@ func StartPortfolioWatcher() {
 func GetPortfolio() *Base {
 	return &Portfolio
 }
+
+// IsExchangeSupported checks if exchange is supported by portfolio address
+func IsExchangeSupported(exchange, address string) (ret bool) {
+	for x := range Portfolio.Addresses {
+		if Portfolio.Addresses[x].Address != address {
+			continue
+		}
+		exchangeList := strings.Split(Portfolio.Addresses[x].SupportedExchanges, ",")
+		if common.StringDataContainsInsensitive(exchangeList, exchange) {
+			return true
+		}
+	}
+	return
+}
+
+// IsColdStorage checks if address is a cold storage wallet
+func IsColdStorage(address string) (ret bool) {
```

Comment:
Just a suggestion dont need to add it, but could return an error as well if address string not found in list. And to isWhitelisted below.

Reply:
at ths point i am going to leave it as just returning a bool its not the job of the method to tell you if the address exists just if it does exist is it flagged 

There are a few other improvements to overall portfolio management that I want to do but that is another PR entirely

Reply:
OKay!